src/app/campaigns/components/campaigns-dashboard/campaigns-dashboard.component.html

```diff
@@ -634,12 +633,12 @@
           color: #1f2337;
         "
       >
-        Adpool SaTT is an innovative tool that allows you to maximize the impact
-        of their marketing campaigns. It allows you to advertise influencers to
-        talk about your brand or product, using blockchain technology. <br />
-        By requesting the creation of an Adpool, our experienced team will
-        contact you and help you set up your campaign. <br />
-        Trust SaTT for a successfulmarketing campaign!
+        {{ 'Wallet_updated_FOOTER1' |  translate }}
+        {{ 'Wallet_updated_FOOTER2' |  translate }}
+        {{ 'Wallet_updated_FOOTER3' |  translate }} <br />
+        {{ 'Wallet_updated_FOOTER4' |  translate }}
+        {{ 'Wallet_updated_FOOTER5' |  translate }} <br />
+        {{ 'Wallet_updated_FOOTER6' |  translate }}
```

Comment:
In order to facilitate localization and internationalization, could we please place this entire block of text into a single i18n translation variable? This will allow us to translate the entire block of text at once, rather than dealing with individual sentences, ensuring a more consistent translation. Thank you.